### DISCUSSION TITLE
Dynamically enable Maska?

### DISCUSSION THREAD

**Discussion by brandonleichty** [beholdr/maska#110] [Q&A] [answered]
Hello! 

I'm wondering if there is a way to dynamically enable disable Maska? 

Doing something like `v-maska="true"` or `v-maska="false"` does't seem to work. 

I currently have a bunch of input fields that are being dynamically rendered with the Vue's dynamic component (`<component :is="" />`), but I don't need "all" of the inputs to have masking. Here's an example: 

```
<component
        :is="component"
        //  v-maska in here somehow 😄 
        v-if="visible"
        v-bind="fieldProps"
        v-on="eventHandlers"
        :modelValue="props[model]"
        @update:modelValue="$emit(`update:${model}`, $event)"
      >
        {{ fieldProps.label }}
</component>
```

Any thoughts / pointers would be greatly appreciated! 🙌

**Comment by beholdr** [comment, marked as answer]
If you set `mask` option to empty string or null it will disable mask, allowing to enter any symbols.
